4.sc_Morphology_Neural_Network_MLP_Model/0.model_training/README.md

```diff
@@ -0,0 +1,28 @@
+# Training Modules
+
+Using `Binary_Classifier_Call.sh` to train the models on a slurm HPC system.
+The PBMC dataset is too large to train on a local machine.
+
+I submit the job via:
```

Comment:
```suggestion
Submit the job via:
```
Consider refraining from speaking in the first person tone, since this is more of an instructional documentation on how to execute a job in a HPC.